ansible/roles/build-grsec-kernel/tasks/compile.yml

```diff
@@ -6,30 +6,36 @@
     creates: "{{ linux_source_directory }}"
 
 - name: apply grsecurity patch
-  shell: patch -p1 < ../{{ grsecurity_patch.filename }}
+  shell: patch -p1 < ../{{ grsecurity_patch.filename | quote }}
   args:
     chdir: "{{ linux_source_directory }}"
 
 - name: put kernel config
   template:
-    src: config-{{ target_os }}
+    src: config-{{ grsecurity_build_strategy }}
```

Comment:
You'll need to also rename config-debian to config-vanilla in the templates directory for this to work. The included configs are critical for now (since they include the configuration of grsec) but maybe we could find a way to automate collecting a base config from the `/boot` directory of the target machine and then automate the running of `make menuconfig` to enable and configure grsec and generate a `.config` file output with the options we want.